src/css/character-grid.css

```diff
@@ -20,13 +20,24 @@
 	grid-template-columns: repeat(auto-fill, var(--character-grid-cell));
 	font-size: var(--character-grid-font-size);
 	flex: 1; /* We don't want both of the elements to take in the space they need.  */
-	min-height: 600px;
-	height: 46rem;
+	min-height: 50vh;
+	height: 50vh;
 	overflow-y: scroll;
 	overflow-x: hidden;
 	margin: 0;
 }
 
+.character-grid::-webkit-scrollbar {
+	-webkit-appearance: none;
```

Comment:
This looks like a Webkit/Chromium-only fix, do you have a solution for Firefox as well?